### DISCUSSION TITLE
GitHub page show 404 on existing file.

### DISCUSSION THREAD

**Discussion by ve3** [community/community#109421] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have GitHub page published [here][apidoclink].  
If I click to the menu **Indices** > **Files**, almost all the files from here can be accessible except **[installer.php][installerlower]** page.  

This file does exists and generated by phpDocumentor 3. It is currently always showing 404 **Page not found**, **File not found**.

However, if I modify the URL manually be replace **[installer][installerlower]** with **[Installer][installerucfirst]** (upper case first letter) then it will be showing fine like [this][installerucfirst].  
(Other pages have all lower case file name if you notice.)

I think this is incorrect behavior and maybe a bug on GitHub page?

[apidoclink]:https://rundizbones.github.io/ModuleAdmin/
[installerlower]:https://rundizbones.github.io/ModuleAdmin/files/installer.html
[installerucfirst]:https://rundizbones.github.io/ModuleAdmin/files/Installer.html

**Comment by NineIT420** [comment]
I understand your issue regarding the accessibility of the phpDocumentor file "installer.php" hosted on your GitHub page. It seems that the file is case sensitive and only displays correctly when the first letter is capitalized. This is not standard behavior and could indeed suggest a bug on the GitHub platform or possibly an issue with the file settings or structure on your page.
As an experienced web developer, my approach to troubleshooting this problem would be to first pull a local copy of the repository to check whether there might have been any error during the generation of the phpDocumentor. If this doesn't reveal any problems, we will then examine the file's permissions, as well as the structure including directories and subdirectories. In the event these investigations are inconclusive, I would suggest notifying GitHub about the potential bug, providing them with all the necessary details they would need to investigate further.

**Reply by ve3** [reply]
I can't find the way to report a bug on GitHub. They link me to here (community forum). There is bug report link but only about security. So, I have no idea what to do about it.

About the errors, there is no error on generate pages and it is accessible correctly on local host.

**Reply by NineIT420** [reply]
It seems like you're facing a challenge concerning issues reporting on GitHub, specifically non-security related bugs, along with an absence of visible errors on the generated pages on your local host.
Approaches you could take:
While submitting a bug on GitHub, consider checking out the repository owner's CONTRIBUTING.md file or similar documentation that may give guideline on how to report errors.

Try running your application in a different environment, if it's working perfectly in your local system, there might be an issue with the deployed environment.

Trace the issue you're encountering and breakdown the feature or function that isn't performing as expected. Maybe recreating the issue with smaller test cases may help highlight what's causing the trouble.

These strategies can help you not only identify the problem but also communicate more effectively with the broader community or support team. Remember, providing detailed and precise information will aid in quicker resolution of the issue.

**Comment by ve3** [comment, marked as answer]
I'm repository owner and there is no **CONTRIBUTING.md**. I don't use it and it doesn't seem to have non-security bug report anywhere except this community forum.

I'll confirm again that this is just HTML pages and it is working on my local host with or without http server (just use file:// protocol). All the pages loaded correctly, no 404 anywhere.

...

While I'm trying to re-produce bug in other path (**/Test**) with just few simple files (means files with link and few word, nothing much in it). It is just working now. All on test path or real path as posted on the question.

The problem is Git, or GitHub on Windows did not detect case sensitive file change. 
This happens by phpDocumentor v2 & 3. When I use v2 of it, it's created **Installer.html** but when I upgraded to v3, it's created **installer.html** instead. All files are correctly named with correct case on my PC but `git`, `github` command did not detect as changed and still use the same name.

Here is test files source https://github.com/RundizBones/rundizbones.github.io/tree/main/Test
Here is test web page https://rundizbones.github.io/Test/
The installer (all lower case) is working as expected now https://rundizbones.github.io/ModuleAdmin/files/installer.html

Fix file case sensitive guide https://stackoverflow.com/a/55541435/128761
To update git config https://stackoverflow.com/a/17688308/128761
